.eslintignore

```diff
@@ -3,3 +3,5 @@ logs/
 firefox/
 lib/vendor.bundle.js
 activity-streams-env/
+lib/SimpleStorage.js
+test/test-SimpleStorage.js
```

Comment:
We pulled these two files directly out from the addon sdk, looks like eslint was not happy with the code style there. Like one-liner if statements, and it threw out a lot of lint errors.